pkg/scheduler/util/utils_test.go

```diff
@@ -17,12 +17,16 @@ limitations under the License.
 package util
 
 import (
+	"fmt"
 	"reflect"
 	"testing"
+	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
s/v1/corev1/?

Reply:
`v1` is automatically added by vscode, and there many `v1 "k8s.io/api/core/v1"` in the project, so I keep it.